app/services/parent_profile_associator.rb

```diff
@@ -17,7 +17,7 @@ def find_parent(profile)
       parents = Profile.where(ref_id: profile.ref_id,
                               benchmark_id: profile.benchmark_id,
                               account_id: nil)
-      raise not_found(profile) unless parents.one?
+      raise not_found(profile) unless parents.any?
```

Comment:
Expecting to find only one profile without a DB constraint on uniqueness is a Bad Idea. We should update this code back to `one?` once we add the constraint for profiles.